BtcMarketsApiClient/BtcMarketsApiClient.Sample/Program.cs

```diff
@@ -5,18 +5,17 @@ namespace BtcMarketsApiClient.Sample
 {
     class Program
     {
-        private const string BaseUrl = "https://api.btcmarkets.net";
-        private const string ApiKey = "add api key here";
-        private const string PrivateKey = "add private key here";
+        private const string BaseUrl = "https://api.btcm.ngin.io";
+        private const string ApiKey = "592e4180-2d44-4fbc-b44d-9acf7c8f0df2";
+        private const string PrivateKey = "eZ60NkcxxHPqkx7UG4AS6h3aCZ6bBSrPUJbT1JvjgN8CKU2DP1xeLbkxc9fG/FMM9wPYstiQXsrlhAcgmKqWDg==";
```

Comment:
please remove API keys and acceptance url fro the sample.

Also squash the commits as I may need to ask my colleagues with .NET to review this Pr.